packages/workspace/src/helpers/const.ts

```diff
@@ -6,9 +6,23 @@ export const configNotLoadedError = 'Workspace configuration can not be loaded';
 
 export const createWorkspaceWrongRequestError = 'createWorkspace has been called with invalid token';
 
-export const bootstrapServiceError = 'Something is wrong while bootstrapping a service';
+export const getLoadingServiceError = (error: Error, serviceMame: string) =>
```

Comment:
```suggestion
export const getLoadingServiceError = (error: Error, serviceName: string) =>
```

Reply:
You have a typo here and below the same for almost every messages

Reply:
thanks!!